tests/integration-tests/conftest.py

```diff
@@ -608,6 +609,11 @@ def inject_additional_config_settings(  # noqa: C901
     with open(cluster_config, encoding="utf-8") as conf_file:
         config_content = yaml.safe_load(conf_file)
 
+    if not dict_has_nested_key(config_content, ("HeadNode", "Ssh", "AllowedIps")):
+        # Permit SSH connection from Jenkins instance only
+        instance_public_ip = get_metadata("public-ipv4")
```

Comment:
what if test is launched from a local machine (not an instance)?

Reply:
Fixed